src/picknik_ur_gazebo_config/objectives/push_button.xml

```diff
@@ -1,47 +0,0 @@
-<?xml version="1.0"?>
-<root BTCPP_format="4" main_tree_to_execute="Push Button">
-    <BehaviorTree ID="Push Button" _description="Push a button with the end effector">
-        <Control ID="Sequence">
-            <SubTree ID="Re-Zero Force-Torque Sensors" />
-            <Action ID="LoadObjectiveParameters" config_file_name="push_along_axis_config.yaml" parameters="{parameters}" />
-            <SubTree ID="CloseGripper" />
-            <Action ID="SaveCurrentState" saved_robot_state="{initial_robot_state}" />
-            <Control ID="Sequence">
-                <Action ID="InitializeMTCTask" task_id="push_button" controller_names="/joint_trajectory_controller" task="{push_button_task}"/>
-                <Action ID="SetupMTCCurrentState" task="{push_button_task}"/>
-                <Action ID="GetPoseFromUser" parameter_name="push_button.target_pose" parameter_value="{target_pose}" />
-                <Action ID="SetupMTCMoveToPose" ik_frame="grasp_link" planning_group_name="manipulator" target_pose="{target_pose}" task="{push_button_task}" use_all_planners="false"/>
-                <Action ID="SetupMTCUpdateGroupCollisionRule" name="AllowGripperCollisionWithOctomap" parameters="{parameters}" task="{push_button_task}" />
-                <Action ID="SetupMTCMoveAlongFrameAxis" task="{push_button_task}" hand_frame="grasp_link" axis_frame="grasp_link" axis_x="0.0" axis_y="0.0" axis_z="1.0" max_distance="0.2" min_distance="0.05" planning_group_name="manipulator" velocity_scale="0.05" acceleration_scale="0.05" />
-                <Action ID="SetupMTCUpdateGroupCollisionRule" name="ForbidGripperCollisionWithOctomap" parameters="{parameters}" task="{push_button_task}" />
-                <Action ID="PlanMTCTask" solution="{full_push_button_solution}" task="{push_button_task}"/>
-                <Action ID="SplitMTCSolution" solution_in="{full_push_button_solution}" index="3" solution_out_1="{move_to_approach_solution}" solution_out_2="{push_solution}" />
-                <Action ID="WaitForUserTrajectoryApproval" solution="{full_push_button_solution}"/>
-                <Action ID="ExecuteMTCTask" solution="{move_to_approach_solution}"/>
-                <Action ID="SaveCurrentState" saved_robot_state="{pre_approach_robot_state}" />
-                <Control ID="Parallel" success_count="1" failure_count="1">
-                    <Action ID="ForceExceedsThreshold" wrench_topic_name="/force_torque_sensor_broadcaster/wrench" hand_frame_name="grasp_link" wrench_frame_name="tool0" minimum_consecutive_wrench_values="10" force_threshold="10.0"/>
```

Comment:
same here

Reply:
I've added a subtree for this, and it seems to work! We could even do subtree port remapping to just have 1 version of this subtree lying around, but decided to not go that route until subtree port remapping is a little more fleshed out unless we want it now.